apps/antalmanac/src/stores/calendarizeHelpers.ts

```diff
@@ -139,43 +130,6 @@ export const calendarizeCustomEvents = (currentCustomEvents: RepeatingCustomEven
     return customEventsInCalendar;
```

Comment:
```suggestion
export function calendarizeCustomEvents(currentCustomEvents: RepeatingCustomEvent[] = []): CustomEvent[] {
    return currentCustomEvents.flatMap((customEvent) => {
        return customEvent.days.filter(Boolean).map((_day, index) => {
            const startHour = parseInt(customEvent.start.slice(0, 2), 10);
            const startMin = parseInt(customEvent.start.slice(3, 5), 10);
            const endHour = parseInt(customEvent.end.slice(0, 2), 10);
            const endMin = parseInt(customEvent.end.slice(3, 5), 10);

            return {
                customEventID: customEvent.customEventID,
                color: customEvent.color ?? '#000000',
                start: new Date(2018, 0, index, startHour, startMin),
                isCustomEvent: true,
                end: new Date(2018, 0, index, endHour, endMin),
                title: customEvent.title,
            };
        });
    });
}
```

This isn't on the diff, but this function should be implemented like this